src/javascript/app/base/footer.js

```diff
@@ -25,6 +28,10 @@ const Footer = (() => {
         $('#footer-regulatory .eu-only').setVisibility(should_show_warning);
     };
 
+    const showAgeRestrictionSign = (should_show_age_restriction_sign) => {
+        $('#footer-regulatory .age-restriction').setVisibility(should_show_age_restriction_sign);
```

Comment:
does it need to be a classname? if we change it to an id, we can easily use javascript instead of jQuery for it

Reply:
I prefer to not do that because we're using it twice in one file